modules/stochastic_tools/test/tests/distributions/normal_direct_type_error.i

```diff
@@ -0,0 +1,43 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 1
+  ny = 1
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Distributions]
+  [this_is_the_wrong_type]
+    type = UniformDistribution
```

Comment:
should be ``TruncatedUniformDistribution``?

Reply:
It doesn't matter, it just needs to be something that is not a normal distribution to test the error message.

Reply:
My bad. I thought you had ``NormalDistribution`` there.